hl7apy/v2_5_1/fields.py

```diff
@@ -1037,9 +1037,12 @@
     'OBX_17': ('sequence', DATATYPES_STRUCTS['CE'], 'CE', 'OBSERVATION_METHOD', None, -1),
     'OBX_18': ('sequence', DATATYPES_STRUCTS['EI'], 'EI', 'EQUIPMENT_INSTANCE_IDENTIFIER', None, -1),
     'OBX_19': ('sequence', DATATYPES_STRUCTS['TS'], 'TS', 'DATE_TIME_OF_THE_ANALYSIS', None, -1),
-    'OBX_20': ('sequence', DATATYPES_STRUCTS['XON'], 'XON', 'PERFORMING_ORGANIZATION_NAME', None, -1),
```

Comment:
Please delete the definition of OBX_20, OBX_21, OBX_22 as they are not defined in the standard